src/projects/project-pull-request-list-item.html

```diff
@@ -86,13 +86,18 @@
         issue: Object,
         showStatus: Boolean,
         selected: Boolean,
+        statuses: Array,
         _cistatus: {
           type: String,
           computed: '_getCIStatus(statuses)'
         },
         _updatedAtString: {
           type: String,
           computed: 'toDateString(pr.updated_at)'
+        },
+        _latestStatusUrl: {
+          type: String,
+          computed: '_getLatestStatusUrl(statuses)'
```

Comment:
This url should be based on the `_cistatus`. Probably we should return `latestci` in `_getCIStatus` and then modify the binding in the class above. You should then be able to use the url that is in this object inside the `<a>` too.